### DISCUSSION TITLE
[2기-A] 최정원

### DISCUSSION THREAD

**Discussion by jung9156** [super-corini/programmers_kdt_II#378] [2기 Monthly Project I]
역대 로또 번호들의 빈도수를 시각화 했고, 빈도수 상위, 하위 번호들을 원하는 만큼 뽑아 랜덤으로 추천해봤습니다.
endpoint : http://ec2-3-37-89-143.ap-northeast-2.compute.amazonaws.com/
github : https://github.com/jung9156/edamonthly_grepp

배포를 해보면서, AWS사용법과 배포에 사용한 nginx, wsgi등에 대해 조금은 알게 된 것 같습니다.

**Comment by jung9156** [comment]
언제까지 AWS를 켜놓으면 될까요? 과금이 두렵습니다! ㅎㅎ

**Reply by super-corini** [reply]
확인했습니다! 다른분들이 확인하실 수 있도록 오늘 자정까지 켜두시는건 어떨까요? :)
